### PR TITLE
[FIX] mass_mailing: image width beyond container

### PR DESCRIPTION
**Pull request by maad-odoo**
Before this commit:
When attempting to increase the width of an image using the 'o_handle' arrows, 
the width keeps on increasing until the mouse arrow leaves the body, resulting 
in an increased image width.

After this commit:
When attempting to increase the width of the image using the 'o_handle' arrows, 
the width increases only up until the closest 'div' tag, resulting in an 
undesired restriction in the image width increase.

task-3593008

### CODE REVIEW

File: addons/mass_mailing/static/src/js/mass_mailing_snippets.js [hunk 1]
```diff
@@ -51,8 +51,10 @@ options.registry.mass_mailing_sizing_x = options.Class.extend({
                     self.change_width(event, self.$target, target_width, offset, false);
                     self.change_width(event, self.$target.prev(), sib_width, sib_offset, true);
                 }
-                if (compass === 'image') {
-                    self.change_width(event, self.$target, target_width, offset, true);
+                if (compass === "image") {
```
Comment (Zynton) [inline addons/mass_mailing/static/src/js/mass_mailing_snippets.js:54]: Unnecessary diff.

File: addons/mass_mailing/static/src/js/mass_mailing_snippets.js [hunk 2]
```diff
@@ -51,8 +51,10 @@ options.registry.mass_mailing_sizing_x = options.Class.extend({
...
+                    let maxWidth = self.$target.closest("div").width();
```
Comment (Zynton) [inline addons/mass_mailing/static/src/js/mass_mailing_snippets.js:55]:
```suggestion
                    const maxWidth = self.$target.closest("div").width();
```

File: addons/mass_mailing/static/src/js/mass_mailing_snippets.js [hunk 3]
```diff
@@ -51,8 +51,10 @@ options.registry.mass_mailing_sizing_x = options.Class.extend({
...
+                    self.$target.css("width", Math.min(maxWidth, Math.round(event.pageX - offset)));
```
Comment (Zynton) [inline addons/mass_mailing/static/src/js/mass_mailing_snippets.js:56]: I suggest this comment to precede it:
```js
// Equivalent to `self.change_width` but ensuring `maxWidth` is the maximum:
```

**Review by Zynton — CHANGES_REQUESTED** [review]

**Comment by maad-odoo** [comment]
@Zynton Changes are done. Please have a look.

**Comment by Zynton** [comment]
@robodoo r+

**Pull request closed without merging**